test/ManualValidation.md

```diff
@@ -0,0 +1,12 @@
+# Manual Validation
+Our goal is to automate as much testing as possible with unit and integration tests, please see [Test.bats](https://github.com/deislabs/ratify/blob/main/test/bats/test.bats) script to review end to end scenario tested today.  While we are working on improving our coverage and sorting out cloud subscriptions account to use for testing, here is the list of scenario that currently requires manual validation.  
```

Comment:
This looks good overall but has a focus on the K8s experience.  IMO - we should have a matrix for K8s versus binary.

Reply:
Ah yes, the cli is not covered by the e2e test. Created issue https://github.com/deislabs/ratify/issues/283. Cli is a very thin layer with sharing same code path as in cluster experience, i have added a note to use a combination of cluster and local. All verifiers can be run through binary, docker auth provider also runs locally.